src/utils/functions.ts

```diff
@@ -0,0 +1,14 @@
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (todos: Todo[], filter: FilterType): Todo[] => {
```

Comment:
Consider adding a comment above the `getFilteredTodos` function to describe its purpose and usage. 📝